### PR TITLE
Add hotfix banner

### PR DESCRIPTION
**Pull request by stevehook**
Display a big ugly warning banner if the `hotfix` branch is present. (It should be created at the start of the hotfixing process and automatically removed when merged to `master`).

image

### CODE REVIEW

File: lib/diff.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Diff
   def self.pull_requests_between(to_sha, from_sha)
-    compare = GitHub.client.compare('DFE-Digital/apply-for-postgraduate-teacher-training', from_sha, to_sha)
+    compare = GitHub.client.compare(GitHub::GITHUB_REPO, from_sha, to_sha)
```
Comment (duncanjbrown) [inline lib/diff.rb:3]: 👌

File: lib/state.rb [hunk 2]
```diff
@@ -51,6 +51,12 @@ def deployers_for_today
     Deployers.for_today
   end
 
+  def hotfix_in_progress?
+    !GitHub.client.branch(GitHub::GITHUB_REPO, GitHub::HOTFIX_BRANCH).nil?
```
Comment (duncanjbrown) [inline lib/state.rb]: At this point I think I'd include `ActiveSupport` and call `.present?`?

File: spec/lib/state_spec.rb [hunk 3]
```diff
@@ -22,12 +22,34 @@
 
   context 'a deploy to production in progress' do
     it 'reports the correct data' do
-      VCR.use_cassette("deploy-to-production-in-progress") do
+      VCR.use_cassette('deploy-to-production-in-progress') do
```
Comment (duncanjbrown) [inline spec/lib/state_spec.rb:25]: Do we rubocop? Maybe we should

**Review by duncanjbrown — APPROVED** [review]
LGTM!

**Comment by duncanjbrown** [comment]
Maybe use a different emoji for `Hotfix in progress` to distinguish from `Master is broken`? 🔥?🚒?

File: spec/lib/state_spec.rb (hunk 3 above)
Comment (stevehook) [reply spec/lib/state_spec.rb:25]: yes good call, I think a separate PR?

image

File: lib/state.rb (hunk 2 above)
Comment (stevehook) [reply lib/state.rb]: yes! that's wrote I wrote originally before remembering its not core ruby :)

**Review by duncanjbrown — APPROVED** [review]

**Pull request merged**